src/str.js

```diff
@@ -204,6 +204,21 @@ class Str {
     return this.value.length
   }
 
+  /**
+   * Returns the first limit characters and ends the limited string with end.
+   *
+   *
+   * @param {Number} limit
+   * @param {String} end
+   *
+   * @returns {Str}
+   */
+  limit (limit = 0, end = '') {
+    return new Str(
+      this.value.substr(0, limit).concat(end)

```

Comment:
Can you please use the `.substring()` method? The `substr` method is deprecated.

Reply:
Ok, switched to `substring`

Reply:
Thank you!